src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -44,15 +45,19 @@ export const MapRoom: React.FC<MapRoomProps> = ({
 
   const containerClasses = classNames("maproom", {
     "maproom--unclickable": isUnclickable,
+    "maproom--iframe": isIframe,
     "maproom--always-show-label":
       !isUnclickable &&
+      !isIframe &&
       (venue.roomVisibility === RoomVisibility.nameCount ||
         (venue.roomVisibility === RoomVisibility.count && hasRecentRoomUsers)),
   });
 
   const titleClasses = classNames("maproom__title", {
     "maproom__title--count":
-      !isUnclickable && venue.roomVisibility === RoomVisibility.count,
+      !isUnclickable &&
+      !isIframe &&
```

Comment:
Using the `isNonInteractableRoom` pattern mentioned in another comment in this review:

```suggestion
      !isNonInteractableRoom &&
```

Reply:
Though.. another question here actually.. do we want to be able to have a 'room title' shown for the `iframe` embedded on the map?

This is probably a @sofisparkle / @sparkletown/product question.

Reply:
For what it's worth, I have a few requests where title wouldn't be necessary

Reply:
Makes sense. Originally my thinking was that this can already be controlled by the venue settings, but since this applies to all 'rooms' in the map, it's not really the same concept.

For now let's leave it as not having a name, and we can address that in future if we find we have a need for it to be otherwise.